crowdin/messages.json

```diff
@@ -1449,6 +1448,7 @@
   "1920861305": "Deriv MT5 is the MT5 platform on Deriv. It is a multi-asset online platform designed to give new and experienced traders access to a wide range of financial markets.",
   "1922529883": "Boom 1000 Index",
   "1923530212": "Flexible.",
+  "1924997360": "[ { 10",
```

Comment:
would you please check this? this is not a valid text to translate
also please make sure that the other one that is removed is correct.

Reply:
Not sure about this
I didn't change any texts inside this PR